src/sentry/api/endpoints/organization_sdk_updates.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import
+
+from itertools import groupby, chain
+from datetime import timedelta
+
+from packaging import version
+from django.utils import timezone
+from rest_framework.response import Response
+from sentry.api.serializers.base import serialize
+
+from sentry.sdk_updates import SdkIndexState, SdkSetupState, get_suggested_updates
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.utils.compat import map
+from sentry.snuba import discover
+
+
+def by_sdk_name(sdk):
+    return sdk["sdk.name"]
+
+
+def by_project_id(sdk):
+    return sdk["project.id"]
+
+
+def serialize(data, projects):
+    # Build datastructure of the latest version of each SDK in use for each
+    # project we have events for.
+    sdks_by_project = [
+        (
+            project_id,
+            [
+                {
+                    "name": sdk_name,
+                    "version": sorted((s["sdk.version"] for s in sdks), key=version.Version).pop(),
```

Comment:
mega nit: It's faster to find the max of a list by iterating through it once rather than sorting.

Reply:
good call